src/App.vue

```diff
@@ -105,11 +105,13 @@
 
 <script>
 
- import router from "./router/index.js";
+import "@fontsource/roboto";
 
- import diary_manager from "@/diary_manager.js";
+import router from "./router/index.js";
 
- import { PROJECT_URL } from "@/constants.js";
+import diary_manager from "@/diary_manager.js";
+
+import { PROJECT_URL } from "@/constants.js";
```

Comment:
a) if you want to make whitespace changes associated with a line, put it in a separate commit - right now I can't easily tell what this is trying to achieve
b) either use the same indentation as the surrounding lines, or introduce a PR that changes it across the project (preferably) or file (at minimum).  Consistency is more important than anything
c) discussions about indentation tend to get quite serious, so if this is something you feel strongly about, you might want to create an issue before spending too much time on a specific solution

Reply:
a) Good point
b) / c) I'll respond to this in a separate discussion.